src/DependencyInjection/HttplugExtension.php

```diff
@@ -94,6 +103,14 @@ public function load(array $configs, ContainerBuilder $container)
             $container->removeAlias(HttpAsyncClient::class);
             $container->removeAlias(HttpClient::class);
         }
+
+        if ($this->useVcrPlugin) {
```

Comment:
unless i miss something, we don't set this property to another value. it should be a check whether the 'vcr' configuration is enabled in the resolved config.

Reply:
Fixed in `03ccd71`.

Reply:
ah, and we need to do it here because the plugin can be configured for several clients, but we only want to do this thing once.